utils/config.ts

```diff
@@ -11,4 +11,14 @@ const schema = z.object({
   REDIS_URL: z.string().default("redis://localhost:6379"),
 });
 
-export const config = schema.parse(process.env);
```

Comment:
Instead of creating a new configuration variable, it would be better to use the `transform` function from Zod.

```ts
import dotenv from "dotenv";
import { z } from "zod";

dotenv.config();

const schema = z.object({
  NODE_ENV: z
    .enum(["test", "development", "production"])
    .default("development"),
  SESSION_DATABASE: z.enum(["in-memory", "redis"]).default("in-memory"),
  REDIS_URL: z.string().default("redis://localhost:6379"),
}).transform((env) => {
  const redisUrl = new URL(env.REDIS_URL);
  return {
    ...env,
    REDIS_HOST: redisUrl.hostname || "localhost",
    REDIS_PORT: z.coerce.number().parse(redisUrl.port || 6379),
  };
});

export const config = schema.parse(process.env);
```

Reply:
fixed, thanks